src/main/java/com/iridium/iridiumskyblock/managers/tablemanagers/TableManager.java

```diff
@@ -62,6 +59,30 @@ public void save() {
             exception.printStackTrace();
         }
     }
+    
+    public void saveHashMap(LinkedHashMap<?, T> uuidLinkedHashMap) {
+        this.saveHashMap(uuidLinkedHashMap, true);
+    }
+    public void saveHashMap(LinkedHashMap<?, T> uuidLinkedHashMap, boolean useChanged) {
+        List<T> tList = new LinkedList<>(uuidLinkedHashMap.values());
+        int sizeList = tList.size();
+        try {
+            for (int i = 0; i < sizeList; i++) {
+                T t = tList.get(i);
+                if (!useChanged) { // all save
```

Comment:
What exactly is the use case for this toggle?

Reply:
For save user data